views/js/runner/proxy.js

```diff
@@ -43,6 +43,8 @@ define([
      * @returns {proxy} - The proxy instance, bound to the selected proxy adapter
      */
     function proxyFactory(proxyName, config) {
+        var proxy, delegateProxy, communicator, communicatorPromise;
+        var holder;
```

Comment:
Why not call this variable `dataHolder` or, event better, `testDataHolder` ? `holder` could be anything and does not say much to someone new to the code and the concepts at stake here. We could also need later a dataHolder for something else than the test data.